tools/eviction/fill_db.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
```

Comment:
still missing the explanation either here on in the readme.

Reply:
documentation added